framework/test/unit/modules/interoperability/mainchain/commands/recover_message.spec.ts

```diff
@@ -59,16 +59,18 @@ import {
 import { CcmSendSuccessEvent } from '../../../../../../src/modules/interoperability/events/ccm_send_success';
 import { InvalidRMTVerification } from '../../../../../../src/modules/interoperability/events/invalid_rmt_verification';
 
-const getHeight = (size: number) => Math.ceil(Math.log2(size)) + 1;
-
-const appendPrecedingToIndices = (indices: number[], terminatedChainOutboxSize: number) => {
-	const mostSignificantBit = 2 ** terminatedChainOutboxSize;
-	return indices.map(index => index + mostSignificantBit);
-};
-
 describe('MessageRecoveryCommand', () => {
 	const interopModule = new MainchainInteroperabilityModule();
 	const leafPrefix = Buffer.from([0]);
+
+	// Choose the index such that the position of the ccm in the outbox tree is at terminatedChainOutboxSize,
+	// i.e. just outside of the tree. Note that a 1 has to be prepended to the binary representation of the index, which is done by adding 2**indexLength.
```

Comment:
> which is done by adding 2**indexLength.

the current code does not use the name `indexLength`.

Reply:
Comments moved to L324 and it has been renamed to `index.length`